spec/controllers/api/encryptables_transfer_controller_spec.rb

```diff
@@ -75,4 +76,64 @@
     end
   end
 
+  context 'Encryptable Credentials Transfer' do
+
+    it 'Does not send encryptable credentials to non existing user' do
+      login_as(:bob)
+
+      request_params = {
+        receiver_id: 139082421,
+        encryptable_id: credentials2.id
+      }
+
+      expect do
+        post :create, params: request_params, xhr: true
+      end.to raise_error(ActiveRecord::RecordNotFound, "User with id 139082421 does not exist")
```

Comment:
make sure correct http code is returned, and also json error message so the frontend can handle it.